samples/sensor/stm32_vbat_sensor/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  description: Usage of STM32 Vbat internal sensor
+  name: stm32_vbat_sensor
+tests:
+  sample.sensor.stm32_vbat_sensor:
+    depends_on: adc
+    tags: sensors
+    build_only: true
+    platform_allow: nucleo_g071rb
```

Comment:
Should it be build only ?
Can't we add a harness that checks voltage is valid ?
```
    harness: console
    timeout: 10
    harness_config:
      type: one_line
      regex:
        - "Current Vbat voltage: 33(.0) V"
```